modules/virt-virtctl-commands.adoc

```diff
@@ -174,6 +174,24 @@ Optional:
 
 |`virtctl vmexport download <vmexport_name> --<vm\|snapshot\|pvc>=<object_name> --output=<output_file> --volume=<volume_name>`
 |Create a `VirtualMachineExport` CR and then download the volume defined in the CR.
+
+|`virtctl vmexport download export --manifest`
+|Retrieve the manifest for an existing export. The manifest does not include the header secret.
+
+|`virtctl vmexport download export --manifest --vm=example`
+|Create a VM export for a VM example, and retrieve the manifest. The manifest does not include the header secret.
+
+|`virtctl vmexport download export --manifest --snap=example`
+|Create a VM export for a VM snapshot example, and retrieve the manifest. The manifest does not include the header secret.
+
+|`virtctl vmexport download export --manifest --include-secret`
+|Retrieve the manifest for an existing export. The manifest includes the header secret.
+
+|`virtctl vmexport download export --manifest --manifest-output-format=json`
+|Retrieve the manifest for an existing export in json format. The manifest does not include the header secret.
+
+|`virtctl vmexport download export --manifest --include-secret --output=manifest.yaml`
+|Retrieve the manifest for an existing export. The manifest will include the header secret and write it to the file specified.
```

Comment:
```suggestion
|Retrieve the manifest for an existing export. The manifest includes the header secret and writes it to the file specified.
```
If the manifest will include the header secret some time in the future, then ignore this suggestion. However,, the suggestion uses the active voice like the instance on line 188.